src/idl_gen_general.cpp

```diff
@@ -1065,11 +1065,9 @@ void GenStruct(StructDef &struct_def, std::string *code_ptr) {
       }
     }
   // generate object accessors if is nested_flatbuffer
+  if (field.nested_flatbuffer != nullptr) {
   auto nested = field.attributes.Lookup("nested_flatbuffer");
```

Comment:
Ah, sorry. Now I understand what you mean. You were referring to line 1069, and not the highlighted line above.

With more context: the nested type already hold by field.nested_flatbuffer is looked up again.
```
  if (field.nested_flatbuffer != nullptr) {
    auto nested = field.attributes.Lookup("nested_flatbuffer");
    auto nested_type = nested->type.struct_def;
    auto nested_type_name = WrapInNameSpace(*nested_type);
```

Obviously, this can be shortened to:
```
  if (field.nested_flatbuffer) {
    auto nested_type_name = WrapInNameSpace(*field.nested_flatbuffer);
```

Ok, I will test this before updating this pull request.



Btw, just below, I see some variables that are `camelCase` and not `snake_case`, namely
`nestedMethodName` and `getNestedMethodName`. They seem to be in this form on purpose. Should I just rename them while I'm at it?

```
   auto nestedMethodName = MakeCamel(field.name, lang_.first_camel_upper)
      + "As" + nested_type_name;
    auto getNestedMethodName = nestedMethodName;
    if (lang_.language == IDLOptions::kCSharp) {
      getNestedMethodName = "Get" + nestedMethodName;
```